linker/shared/src/main/scala/org/scalajs/linker/standard/LinkedClass.scala

```diff
@@ -112,9 +112,15 @@ final class LinkedClass(
   }
 
   private[linker] def optimized(
-      methods: List[Versioned[MethodDef]]
+      methods: List[Versioned[MethodDef]],
+      exportedMembers: List[Versioned[JSMethodPropDef]],
+      jsConstructorDef: Option[Versioned[JSConstructorDef]]
   ): LinkedClass = {
-    copy(methods = methods)
+    copy(
+        methods = methods,
+        exportedMembers = exportedMembers,
+        jsConstructorDef = jsConstructorDef
+    )
```

Comment:
I guess this should now be in the new style, with a trailing comma:
```suggestion
    copy(
      methods = methods,
      exportedMembers = exportedMembers,
      jsConstructorDef = jsConstructorDef,
    )
```

Reply:
Scalastyle still fails on trailing commas :(

Reply:
Ouch.